client/lib/analytics/index.js

```diff
@@ -629,18 +641,21 @@ const analytics = {
 		initialized: false,
 
 		initialize: function() {
-			const parameters = {};
 			if ( ! analytics.ga.initialized ) {
-				if ( _user && _user.get() ) {
-					parameters.userId = hashPii( _user.get().ID );
-				}
-				window.ga( 'create', config( 'google_analytics_key' ), 'auto', parameters );
-				window.ga( function( tracker ) {
-					const clientId = tracker.get( 'clientId' );
-					window.ga( 'set', 'dimension3', clientId );
-				} );
-				window.ga( 'set', 'anonymizeIp', true );
-				window.ga( 'set', 'useAmpClientId', true );
+				const parameters = {
+					anonymize_ip: true,
```

Comment:
I think we should be setting `send_page_view: false,` in the initialization here.
https://developers.google.com/analytics/devguides/collection/gtagjs/#disable_pageview_measurment